src/app/register/components/form-personal-additional-emails/form-personal-additional-emails.component.html

```diff
@@ -0,0 +1,101 @@
+<!-- ADDITIONAL EMAILS -->
+
+<ng-container [formGroup]="additionalEmails">
+  <div
+    class="input-container"
+    *ngFor="
+      let additionalEmailControl of additionalEmails.controls | keyvalue;
+      index as i
+    "
+  >
+    <mat-form-field appearance="outline">
+      <mat-label>
+        <ng-container i18n="@@register.additionalEmails"
+          >Additional email</ng-container
+        >
+        {{
+          additionalEmailControl.key !== '0'
+            ? parseInt(additionalEmailControl.key)
+            : ''
+        }}
+        <ng-container i18n="@@register.optional">(Optional)</ng-container>
+      </mat-label>
+      <input
+        formControlName="{{ additionalEmailControl.key }}"
+        matInput
+        [errorStateMatcher]="backendErrorsMatcher"
+      />
+      <mat-error
+        *ngIf="
+          additionalEmails.hasError('email', additionalEmailControl.key) ||
+          additionalEmails.hasError('pattern', additionalEmailControl.key)
+        "
+        i18n="@@register.invalidEmail"
+      >
+        Invalid email format
+      </mat-error>
+      <!--
+        <mat-error
+          *ngFor="
+            let error of getControlErrorAtFormLevel(
+              additionalEmailControl.value,
+              'allEmailsAreUnique'
+            )
+          "
+          i18n="@@register.additionalEmailsCanBeDuplicated"
+        >
+          Additional email cannot match primary email
+        </mat-error>
+      -->
+      <mat-error
+        *ngFor="
+          let error of getControlErrorAtFormLevel(
+            additionalEmailControl.value,
+            'backendErrors'
+          )
+        "
+      >
+        <app-backend-error [errorCode]="error"></app-backend-error>
+      </mat-error>
+    </mat-form-field>
+
+    <button
+      *ngIf="i == 0"
+      mat-icon-button
+      aria-label="info about names"
+      type="button"
+      [mdePopoverTriggerFor]="namesPopover"
+      mdePopoverTriggerOn="click"
+      #namesPopoverTrigger="mdePopoverTrigger"
+    >
+      <mat-icon>info</mat-icon>
+    </button>
+  </div>
+</ng-container>
+
+<a (click)="addAdditionalEmail()" class="mat-button-font">
```

Comment:
@leomendoza123 I think there should as well an option to remove the additional emails in case the user changes his mind

Reply:
That makes sense, with this PR I am just cloning current functionality, where they can just leave the input empty if they do not want to add additional emails. 

Let's ask Dam about this at our next meeting.

Reply:
oh wait, the old app also have this delete button, I will add it 👍